discord/colour.py

```diff
@@ -117,6 +118,11 @@ def default(cls):
         """A factory method that returns a :class:`Colour` with a value of ``0``."""
         return cls(0)
 
+    @classmethod
+    def random(cls):
+        """A factory method that returns a :class:`Colour` with a random value."""
```

Comment:
This probably doesn't matter and would only be a point of clarity and preventing future pedantry, but having this say that it returns a color with a random hue (and either saying nothing of the saturation or value, or saying that those are both maxed in this) rather than using the word value which has specific meaning in discussing colors different to the use here may make some sense. (even though most people would realize that a random value without mentioning hue is a much less likely interpretation)